### DISCUSSION TITLE
xmake是否能够打包已经编译好的库？

### DISCUSSION THREAD

**Discussion by memorelic** [xmake-io/xmake#4039] [Q&A]
本地有一些很早之前使用MSVC管理和编译的库，具体结构大致如下：
xxlib：
   - bin：
      - Debug：存放Debug模式编译的动态库dll和exe
      - Release：存放Release模式编译的动态库dll和exe
   - include：存放改库的头文件
   - lib：
     - Debug：存放Debug模式动态库的lib
     - Release：存放Release模式动态库的lib
     
在不用xmake重新编译该库的情况下，能否将其打包，使得在其它新工程中能用add_requires("xxxlib")来添加该库依赖？
或者是否有其它的管理方法管理这种库呢？

**Comment by star-hengxing** [comment]
package + on_fetch

**Reply by memorelic** [reply]
我照着文档试了一下xmake package，它提示我是否自动生成xmake.lua后生成失败。
您的意思是我自己按照规则写一个xmake.lua后再用package打包吗？

**Reply by star-hengxing** [reply]
参考一下这个
https://github.com/star-hengxing/lajolla_public/blob/main/xmake.lua

**Reply by memorelic** [reply]
好的，我明白了，谢谢您的解答。
